docs/guides/algorithmiq-tem.ipynb

```diff
@@ -165,70 +167,145 @@
     "    qc.u(-1.2, -1.2, 0.3, 2)\n",
     "\n",
     "# Define the observables\n",
-    "observable = SparsePauliOp(\"YXZ\", 1.0)\n",
+    "observable = SparsePauliOp(\"IYX\", 1.0)\n",
     "\n",
     "# Define the execution options\n",
     "service = QiskitRuntimeService()\n",
     "backend_name = service.least_busy(operational=True).name\n",
+    "\n",
     "instance = \"<IQP_HUB/IQP_GROUP/IQP_PROJECT>\"\n",
     "\n",
-    "pub = (qc, observable)\n",
+    "pub = (qc, [observable])\n",
     "options = {\n",
-    "    \"backend_name\": backend_name,\n",
-    "    \"num_max_shots\": 100,\n",
+    "    \"max_execution_time\": 60,\n",
     "}\n",
-    "job = tem.run(instance=instance, pub=pub, options=options)"
+    "job = tem.run(pubs=[pub], instance=instance, backend_name=backend_name, options=options)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "03998691",
+   "id": "6e2a97e2",
    "metadata": {},
    "source": [
-    "The following code checks your Qiskit Function workload's status or return results:"
+    "Use the Qiskit Serverless APIs to check your Qiskit Function workload's status:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "856fe992",
+   "id": "47b0db71",
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(job.status())\n",
-    "result = job.result()"
+    "print(job.status())"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "e2817b13",
+   "id": "fbaeace8",
    "metadata": {},
    "source": [
-    "## Advanced options\n",
-    "\n",
-    "You can fine-tune your calculations and get more verbose information by passing additional options to TEM."
+    "You can return the results as:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "id": "c2ce65e3",
+   "execution_count": null,
+   "id": "ad5b9a53",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import logging\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5e138d3",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"info\">\n",
+    "  The expected value for the noiseless circuit for the given operator should be around `0.18409094298943401`.\n",
+    "</Admonition>\n",
     "\n",
-    "options = {\n",
-    "        \"backend_name\": backend_name,\n",
-    "        \"num_max_shots\": 1_000,\n",
-    "        \"simulate_with_noise_model\": True,\n",
-    "        \"mitigate_readout_error\": True,\n",
-    "        \"num_readout_calibration_shots\": 10_000,\n",
-    "        \"logging_level\": logging.WARNING\n",
-    "    }\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c6ffb685",
+   "metadata": {},
+   "source": [
+    "## Inputs\n",
+    "\n",
+    "**Parameters**\n",
```

Comment:
It bothers me slightly that the parameters are not all in the same order across functions, even though they are all the same. But it's not a huge deal.